lib/galaxy/tool_util/deps/container_resolvers/mulled.py

```diff
@@ -633,6 +633,14 @@ def pull(self, container):
             cmds = container.build_mulled_singularity_pull_command(
                 cache_directory=self.cache_directory.path, namespace=self.namespace
             )
+            # when Galaxy runs in Docker with uswgi, HOME remains set to /root, this is a workaround for that case
+            homedir = os.environ.get("HOME")
+            if homedir and not (os.access(homedir, os.R_OK) and os.access(homedir, os.X_OK)):
+                galaxy_homedir = os.environ.get("GALAXY_HOME")
+                if galaxy_homedir is not None:
```

Comment:
```suggestion
                if galaxy_homedir:
```

Reply:
thanks, fixed. and I noticed that I had actually removed the piece a key piece of code by mistake, so that's back too.